src/lib/transaction_snark/transaction_snark.ml

```diff
@@ -4581,6 +4581,130 @@ module Make_str (A : Wire_types.Concrete) = struct
       in
       zkapp_command
 
+    module Single_account_update_spec = struct
```

Comment:
Can you add a comment to describe what this is for (as there are a few different specs) and when it can be used?